src/_includes/partials/components/brand.njk

```diff
@@ -1,3 +1,7 @@
             <div class="brand">
-                <a rel="home" href="/"{% if page.url === '/' %} aria-current="page"{% endif %}>Trivet</a>
+                {% set homeUrl = "/" %}
+                {% if lang !== config.defaultLanguage %}
```

Comment:
As we discussed on a call, if the language is not the default, the homepage URL isn't defined, so the homepage link on 404 pages will redirect the user to `/undefined`, which of course returns the 404 page, and so on... :)

```suggestion
                {% if lang and lang !== config.defaultLanguage %}
```